float32 left
float32 right
uint16 wheelMode
